Assets/Mapzen/TileAddress.cs

```diff
@@ -14,16 +14,15 @@ public TileAddress(int x, int y, int z)
 
         public static TileAddress FromLngLat(LngLat lngLat, int zoom)
         {
-            double tileResolution = Geo.EarthCircumferenceMeters / Geo.TilePixelSize;
-            double tileSize = tileResolution / (1 << zoom);
+            double tileSize = Geo.EarthCircumferenceMeters / (1 << zoom);
 
             MercatorMeters meters = Geo.Project(lngLat);
 
             double px = meters.x / tileSize;
             double py = meters.y / tileSize;
 
-            int tileX = (int)(Math.Ceiling(px / Geo.TilePixelSize) - 1);
-            int tileY = (int)(Math.Ceiling(py / Geo.TilePixelSize) - 1);
+            int tileX = (int)(Math.Ceiling(px) - 1);
```

Comment:
Let's say someone is scanning tiles near the international dateline and their coordinates go to -181 degrees longitude. `Geo.Project()` would produce negative mercator meters in x and therefore the resulting tile address would also be negative in x. 

My gut reaction is that our utilities shouldn't produce a negative tile address, I'm just wondering where to perform the wrapping step.